modules/openapi-generator/src/main/resources/erlang-proper/api.mustache

```diff
@@ -0,0 +1,27 @@
+-module({{classname}}_api).
+
+-export([ {{#operations}}{{#operation}}{{^-first}}
+        , {{/-first}}{{operationId}}/{{arity}}{{/operation}}{{/operations}}
+        ]).
+
+-define(BASE_URL, "{{{basePathWithoutHost}}}").
+
+{{#operations}}
+{{#operation}}
+%% @doc {{{summary}}}
+{{^notes.isEmpty}}
+%% {{{notes}}}
+{{/notes.isEmpty}}
+-spec {{operationId}}({{#allParams}}{{#required}}{{^-first}}, {{/-first}}{{{dataType}}}{{/required}}{{/allParams}}) ->
+  {{packageName}}_utils:response().
+{{operationId}}({{#allParams}}{{#required}}{{^-first}}, {{/-first}}{{paramName}}{{/required}}{{/allParams}}) ->
+  Method      = {{httpMethod}},
+  Host        = application:get_env({{packageName}}, host, "http://localhost:8080"),
+  Path        = ["{{{replacedPathName}}}"],
+  Body1       = {{^formParams.isEmpty}}{form, [{{#formParams}}{{#required}}{{^-first}}, {{/-first}}{<<"{{baseName}}">>, {{paramName}}}{{/required}}{{/formParams}}]++{{packageName}}_utils:optional_params([{{#formParams}}{{^required}}{{^-first}}, {{/-first}}'{{baseName}}'{{/required}}{{/formParams}}], _OptionalParams)}{{/formParams.isEmpty}}{{#formParams.isEmpty}}{{#bodyParams.isEmpty}}[]{{/bodyParams.isEmpty}}{{^bodyParams.isEmpty}}{{#bodyParams}}{{paramName}}{{/bodyParams}}{{/bodyParams.isEmpty}}{{/formParams.isEmpty}},
```

Comment:
1) Please use `{{{baseName}}}` instead of `{{baseName}}` to avoid special HTML characters being escaped. 

2) Instead of "Body1", what about naming it as "LocalVarHttpBody"?